packages/next-css/css-loader-config.js

```diff
@@ -10,12 +10,12 @@ module.exports = (
   })
   let postcssLoader
 
-  if (postcssConfig) {
+  if (postcssConfig || postcssLoaderOptions.config) {
```

Comment:
Would suggest to use `postcssLoaderOptions` here, then in `next.config.js` you can do:

```
withSass({
  postCssLoaderOptions: {
    sourceMap: true,
    plugins: loder => ([....])
  },
  sassLoaderOptions: {
    sourceMap: true,
  },
})
```

Reply:
I didn't even realize that `plugins` was outside of the config object. Yeah, definitely to change this up.